### PR TITLE
Add layer lotabweichungen

### PR DESCRIPTION
**Pull request by Ariadnigaki**
https://github.com/geoadmin/service-sphinxsearch/pull/289

### CODE REVIEW

**Comment by loicgasser** [comment]
Could you rebase? I think the tests are failing because the layers config has cahnged.

**Comment by Ariadnigaki** [comment]
@loicgasser Everything is up to date

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -2427,3 +2427,24 @@ class SwissBuildings3dPerimeter(Base, ShopStandardClass, Vector):
...
+
+
+class Lotabweichungen(Base, Vector):
+    __tablename__ = 'lotabweichungen_nur_punkte'
+    __table_args__ = ({'schema': 'geodaesie', 'autoload': False})
+    __template__ = 'templates/htmlpopup/lotabweichungen.mako'
+    __bodId__ = 'ch.swisstopo.lotabweichungen'
+    __label__ = 'name'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    name = Column('name', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: Use Unicode instead of Text

File: chsdi/locale/empty_chsdi.po [hunk 2]
```diff
@@ -8860,3 +8860,33 @@ msgstr ""
 
 msgid "klimaeignung_zone"
 msgstr ""
+
+msgid "tt_eta_ch"
```
Comment (loicgasser) [inline chsdi/locale/empty_chsdi.po]: Can you search the features of this layer?
If yes you need to use the convention `layerId.columnName` as IDs for translations.

**Comment by loicgasser** [comment]
can you put the DLL of the table in the PR? (without access stuff)
LGTM otherwise

**Comment by Ariadnigaki** [comment]
[Data link layer](https://mf-geoadmin3.dev.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fdev_ltgar_lotabweichungen&lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.lotabweichungen&X=278827.65&Y=666047.09&zoom=4)

File: chsdi/models/vector/stopo.py [hunk 3]
```diff
@@ -2427,3 +2427,24 @@ class SwissBuildings3dPerimeter(Base, ShopStandardClass, Vector):
...
 register('ch.swisstopo.swissbuildings3d', SwissBuildings3dPerimeter)
+
+
+class Lotabweichungen(Base, Vector):
+    __tablename__ = 'lotabweichungen_nur_punkte'
+    __table_args__ = ({'schema': 'geodaesie', 'autoload': False})
+    __template__ = 'templates/htmlpopup/lotabweichungen.mako'
+    __bodId__ = 'ch.swisstopo.lotabweichungen'
+    __label__ = 'name'
+    id = Column('bgdi_id', Integer, primary_key=True)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: CONSTRAINT lotabweichungen_pkey PRIMARY KEY (id)

in DB, why not use id? It's also an integer. If you don't use ID at least change the pkey definition in DB.

**Comment by loicgasser** [comment]
By DDL I mean:

```
CREATE TABLE geodaesie.lotabweichungen
(
  bgdi_modified timestamp without time zone,
  bgdi_created timestamp without time zone,
  bgdi_modified_by character varying(50),
  bgdi_created_by character varying(50),
  name text,
  land text,
  messjahr integer,
  instrument text,
  xi_ch double precision,
  eta_ch double precision,
  xi_etrf double precision,
  eta_etrf double precision,
  winkel double precision,
  laenge double precision,
  zoomlevel integer,
  symbol integer,
  bgdi_id serial NOT NULL,
  the_geom geometry,
  id integer NOT NULL,
  CONSTRAINT lotabweichungen_pkey PRIMARY KEY (id),
  CONSTRAINT bgdi_geometry_valid_check_the_geom CHECK (st_isvalid(the_geom)),
  CONSTRAINT enforce_dims_the_geom CHECK (st_ndims(the_geom) = 2),
  CONSTRAINT enforce_geotype_the_geom CHECK (geometrytype(the_geom) = 'POINT'::text OR geometrytype(the_geom) = 'LINESTRING'::text OR the_geom IS NULL),
  CONSTRAINT enforce_srid_the_geom CHECK (st_srid(the_geom) = 21781)
)
WITH (
  OIDS=FALSE
);
CREATE UNIQUE INDEX lotabweichungen_bgdi_id_unique
  ON geodaesie.lotabweichungen
  USING btree
  (bgdi_id);

-- Index: geodaesie.lotabweichungen_eta_ch

-- DROP INDEX geodaesie.lotabweichungen_eta_ch;

CREATE INDEX lotabweichungen_eta_ch
  ON geodaesie.lotabweichungen
  USING btree
  (eta_ch);

-- Index: geodaesie.lotabweichungen_eta_etrf

-- DROP INDEX geodaesie.lotabweichungen_eta_etrf;

CREATE INDEX lotabweichungen_eta_etrf
  ON geodaesie.lotabweichungen
  USING btree
  (eta_etrf);

-- Index: geodaesie.lotabweichungen_geomtrytype

-- DROP INDEX geodaesie.lotabweichungen_geomtrytype;

CREATE INDEX lotabweichungen_geomtrytype
  ON geodaesie.lotabweichungen
  USING btree
  (geometrytype(the_geom) COLLATE pg_catalog."default");

CREATE INDEX lotabweichungen_id_unique
  ON geodaesie.lotabweichungen
  USING btree
  (id);

CREATE INDEX lotabweichungen_instrument
  ON geodaesie.lotabweichungen
  USING btree
  (instrument COLLATE pg_catalog."default");

CREATE INDEX lotabweichungen_laenge
  ON geodaesie.lotabweichungen
  USING btree
  (laenge);

CREATE INDEX lotabweichungen_land
  ON geodaesie.lotabweichungen
  USING btree
  (land COLLATE pg_catalog."default");

CREATE INDEX lotabweichungen_messjahr
  ON geodaesie.lotabweichungen
  USING btree
  (messjahr);

CREATE INDEX lotabweichungen_name
  ON geodaesie.lotabweichungen
  USING btree
  (name COLLATE pg_catalog."default");

CREATE INDEX lotabweichungen_symbol
  ON geodaesie.lotabweichungen
  USING btree
  (symbol);

CREATE INDEX lotabweichungen_the_geom
  ON geodaesie.lotabweichungen
  USING gist
  (the_geom);

CREATE INDEX lotabweichungen_winkel
  ON geodaesie.lotabweichungen
  USING btree
  (winkel);

CREATE INDEX lotabweichungen_xi_ch
  ON geodaesie.lotabweichungen
  USING btree
  (xi_ch);

CREATE INDEX lotabweichungen_xi_etrf
  ON geodaesie.lotabweichungen
  USING btree
  (xi_etrf);

CREATE INDEX lotabweichungen_zoomlevel
  ON geodaesie.lotabweichungen
  USING btree
  (zoomlevel);

CREATE TRIGGER update_metaspalten_lotabweichungen
  BEFORE INSERT OR UPDATE
  ON geodaesie.lotabweichungen
  FOR EACH ROW
  EXECUTE PROCEDURE public.setze_lastupdate_firstdate();
```

File: chsdi/locale/empty_chsdi.po (hunk 2 above)
Comment (loicgasser) [inline chsdi/locale/empty_chsdi.po]: Can you answer my question please? Can you search the features in the search field or in the query tool? If you don't know what it is just ask me.

**Comment by loicgasser** [comment]
Also can you rebase please?

**Comment by loicgasser** [comment]
and squash

File: chsdi/locale/empty_chsdi.po (hunk 2 above)
Comment (loicgasser) [inline chsdi/locale/empty_chsdi.po]: @Ariadnigaki -> this was false

**Pull request merged**